remoteworkitem/trackeritem_repository.go

```diff
@@ -67,6 +68,24 @@ func bindAssignees(db *gorm.DB, remoteWorkItem RemoteWorkItem, providerType stri
 	}
 	// copy all fields from remoteworkitem into result workitem
 	for fieldName, fieldValue := range remoteWorkItem.Fields {
+		// creator
+		if fieldName == remoteCreatorLogin {
+			if fieldValue == nil {
+				workItem.Fields[workitem.SystemCreator] = nil
+				continue
+			}
+			creatorLogin := fieldValue.(string)
+			creatorProfileURL := remoteWorkItem.Fields[remoteCreatorProfileURL].(string)
+			identity, err := identityRepository.Lookup(context.Background(), creatorLogin, creatorProfileURL, providerType)
+			if err != nil {
+				return nil, err
```

Comment:
hum, should I really wrap again the the error coming from the identityRepository.Lookup() method ?